README.md

```diff
@@ -69,6 +69,36 @@ result.save("processed/image.png")
 result.save("image.png", overwrite: true) # Careful!
 ```
 
+#### Processing images over 10 megapixels
+
+The Remove.bg API provides a [ZIP format](https://www.remove.bg/api#zip-format)
+which includes all the information required to efficiently composite a large
+image with transparency.
+
+The gem can handle this composition for you, but you'll need
+[ImageMagick]/[GraphicsMagick] or [libvips] available on your computer.
+
+[ImageMagick]: https://www.imagemagick.org
+[GraphicsMagick]: http://www.graphicsmagick.org
+[libvips]: http://libvips.github.io/libvips/
+
+Please configure the image processing library you'd like to use:
+
+```ruby
+RemoveBg.configure do |config|
+  config.image_processor = :minimagick # For ImageMagick or GraphicsMagick
+  # or
+  config.image_processor = :vips
+end
+```
+
+Then process images specifying the `zip` format:
+
+```ruby
+result = RemoveBg.from_file("large-image.jpg", format: "zip")
+result.save("result-with-transparency.png")
```

Comment:
Is there a way to save the raw ZIP file too, if I want to do the compositing outside of the gem?

Reply:
I've added a `#save_zip` method to `CompositeResult`. There isn't the same corresponding method on `Result` (I think that would be confusing).

So for the moment the user needs to either keep track of when they use `format: :zip` or check `result.respond_to?(:save_zip)`

Reply:
Sounds good